### PR TITLE
Adding a tpl file to facilitate theming search results

### LINKED ISSUE #492: Present search results
Need Solr-powered search results page that presents results from both old and new app. This may need to be broken into sub-issues.

### PR DESCRIPTION
**Pull request by blisteringherb**
Fixes #492 

This will need additional issues for styling the search results
and weighting results appropriately.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/search-result.tpl.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <a href="<?php print $url; ?>"><?php print $title; ?></a>
+  </h3>
+  <?php print render($title_suffix); ?>
+  <p><?php print $result['fields']['bundle_name']; ?></p>
+  <div class="search-snippet-info">
+    <?php if ($snippet): ?>
+      <p class="search-snippet"<?php print $content_attributes; ?>><?php print $snippet; ?></p>
+    <?php endif; ?>
+    <?php if ($info): ?>
+      <p class="search-info"><?php //print $info; ?></p>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/search-result.tpl.php:79]: Is this supposed to be commented?

**Comment by barryclark** [comment]
:thumbsup:

**Comment by barryclark** [comment]
Breakdown of all of the variables that are available is really useful, cheers for including that.

**Comment by DFurnes** [comment]
+1000 on the variable listing. That's fantastic.

**Pull request merged**